### ISSUE TITLE
Create examples of OrderBy and Limit

### ISSUE THREAD

**Issue by wovalle** [wovalle/fireorm#54]
### Problem
As soon as features are added, we should provide examples on how to use them to help people using fireorm for the first time.

### Acceptance Criteria
- A new example should be added to /examples implementing OrderBy and Limit
- [README](https://github.com/wovalle/fireorm/blob/master/examples/README.md) is updated with the new example

**Comment by jonesnc** [comment]
@wovalle I'm working on this one (unless someone else already is).

**Comment by wovalle** [comment]
Thanks @jonesnc! Nobody is working on this as far as I know 😄

**Comment by jonesnc** [comment]
@wovalle Since https://github.com/wovalle/fireorm/pull/72 repurposes the `examples`, I think I'll start writing example usage of `orderBy` and `limit` and add it to the README.

Does that sound like a good idea?

**Comment by wovalle** [comment]
Yeah we can do that. I'll also create a integration tests for those.
